app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -1,247 +1,191 @@
-      }
+  legend.label {
+    font-size: inherit;
+    left: 0;
+    position: absolute
+  }
 
-      &.has_many_fields ol {
-        float: left;
-        width: 100%;
-        margin: 0;
-      }
-    }
+  .choices-group,
+  .fragments-group {
+    float: left;
+    padding: 0 0 0 20%;
+    width: 74%;
   }
 
-  /* Text Fields */
-  input[type=text], input[type=password], input[type=email], input[type=number], input[type=url], input[type=tel], textarea {
-    width: calc(80% - #{$text-input-total-padding});
-    border: $border-width solid #c9d0d6;
-    @include rounded;
-    font-size: 0.95em;
-    @include sans-family;
-    outline: none;
-    padding: 8px $text-input-horizontal-padding 7px;
-
-    &:focus {
-      border: $border-width solid #99a2aa;
-      @include shadow(0,0,4px,#99a2aa);
-    }
+  .choice,
+  .fragment {
+    margin: 0 0 0.5em 0;
   }
 
-  fieldset > ol > li {
+  .fragment {
+    display: inline-block;
+    margin: 0 0.5em 0 0;
 
-    /* Hints */
-    p.inline-hints {
-      font-size: 0.95em;
-      font-style: italic;
-      color:#666;
-      margin: 0.5em 0 0 20%;
+    label {
+      display: none;
     }
+  }
 
-    /* Date and Time Fields */
-    &.date_select, &.time_select, &.datetime_select {
-      fieldset ol li {
-        float:left; width:auto; margin:0 0.5em 0 0;
-        label { display: none; }
-        input { display:inline; margin:0; padding:0;  }
-      }
-    }
+  // BOOLEAN OVERRIDES
 
-    /* Check Boxes or Radio fields */
-    &.check_boxes, &.radio {
-      fieldset ol {
-        margin-bottom:-0.6em;
-        li {
-          margin:0.1em 0 0.5em 0;
-          label {
-            float:none;
-            width:100%;
-            input { margin-right:0.2em; }
-          }
-        }
-      }
-    }
+  .boolean input,
+  .choice input {
+    display: inline-block; // for IE
+    margin: 0 0.5em 0 0;
+  }
 
-    /* Boolean Field */
-    &.boolean {
-      height: 1.1em;
-      label {
-        width: 80%;
-        padding-left:20%;
-        padding-right: 10px;
-        text-transform: none !important;
-        font-weight: normal;
-        input { margin:0 0.5em 0 0.2em; }
-      }
-    }
+  .boolean label {
+    padding-left: 20%;
+    padding-right: 10px;
+    width: 80%;
+  }
 
-    /* Hidden fields */
-    &.hidden {
-      padding: 0;
-    }
+  // HINTS and ERRORS
 
-    /* Errors */
-    p.inline-errors {
-      color: $error-color;
-      font-weight: bold;
-      margin:0.3em 0 0 20%;
-    }
-    ul.errors {
-      color: $error-color;
-      margin:0.5em 0 0 20%;
-      list-style:square;
-      li { padding:0; border:none; display:list-item; }
-    }
+  .inline-hints,
+  .inline-errors,
+  .errors {
+    margin: 0.3em 0 0 20%;
+  }
 
-    &.error {
-      input[type=text], input[type=password], input[type=email], input[type=number], input[type=url], input[type=tel], textarea {
-        border: $border-width solid $error-color;
-      }
-    }
+  .inline-hints {
+    color: #999;
```

Comment:
Color literals like `#999` should only be used in variable declarations; they should be referred to via variable everywhere else.